src/components/Path/readme.md

```diff
@@ -0,0 +1,35 @@
+##### Path basic example
+
+```js
+import React from 'react';
+import { Path, PathStep } from 'react-rainbow-components';
+
+const initialState = { currentStep: 'arrived' };
+
+    <div className="rainbow-p-around_x-large rainbow-align-content_center">
+        <Path currentStepName={state.currentStep} onClick={stepName => setState({ currentStep: stepName })}>
+            <PathStep name="scheduled" label="Scheduled" />
+            <PathStep name="in-progress" label="InProgress" />
+            <PathStep name="arrived" label="Arrived" />
+            <PathStep name="delivered" label="Delivered" />
+        </Path>
+    </div>
```

Comment:
for new examples we will not use this anymore, instead make it a functional component with hooks, note that this form the user can't copy and paste